aphrodite/modeling/layers/sampler.py

```diff
@@ -180,25 +180,26 @@ def _get_custom_token_bans(
 
 def _apply_logits_processors(
     logits: torch.Tensor,
-    sampling_metadata: SamplingMetadata,
+    metadata: SamplingMetadata,
```

Comment:
Why the rename?

Reply:
Line length primarily, ease of reading secondarily.
There was no other `metadata` that required the disambiguation of `sampling_`.